packages/perseus/src/components/text-input.jsx

```diff
@@ -15,8 +15,6 @@ type Props = {|
     id?: string,
     placeholder?: string,
     onKeyDown?: () => void,
-
-    // TODO: Remove this
     style?: $FlowFixMe,
```

Comment:
Do you feel up to making this the `StyleType` (as it seems to be passed down directly to the WB component that uses that type for its `style` prop).

https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-form/src/components/text-field.js#L108

Reply:
Good thinking, done.